README.md

```diff
@@ -74,14 +74,13 @@ configure ESLint for Angular projects** especially when migrating from TSLint. [
         files: ['*.ts'],
```

Comment:
Can you update the name on line 71 from `@ni/eslint-config/typescript` to `@ni/eslint-config/angular` too?

Reply:
Done.